docs/concepts/capabilities.mdx

```diff
@@ -20,19 +20,23 @@ Some examples of non-functional requirements you might be familiar with:
 
 For example, when writing a web service that exposes a [RESTful API][restful-api], the HTTP server and the database are not parts of your business logic -- your primary concern is structuring your API so that users can easily get the data they need.
 
-Non-functional requirements are services *used* by your actor to perform business logic. **We call these services "capabilities".**
-
 [restful-api]: https://en.wikipedia.org/wiki/REST
 
-# Capability Contracts
+### Capability Contracts
+
+:::info
+
+The following section was written before WASI Preview 2 was stable. wasmCloud supports the [wasi-cloud-core](https://github.com/WebAssembly/wasi-cloud-core) interfaces, the existing `wasmcloud:*` interfaces, and custom interfaces written by third parties.
```

Comment:
```suggestion
The following section was written before WASI Preview 2 was stable. wasmCloud supports many [wasi-cloud-core](https://github.com/WebAssembly/wasi-cloud-core) interfaces, the existing `wasmcloud:*` interfaces, and custom interfaces written by third parties.
```

NIT: We don't support *all* of them just yet, right?

I'm fine with the current wording, but "many" might be a bit more accurate than "the". Or maybe just delete the "the".